c2corg_api/models/image.py

```diff
@@ -1,32 +1,63 @@
 from sqlalchemy import (
+    Boolean,
     Column,
+    Float,
+    ForeignKey,
     Integer,
     SmallInteger,
-    ForeignKey,
-    Enum
+    String
     )
 
 from colanderalchemy import SQLAlchemySchemaNode
 
-from c2corg_api.models import schema
-from c2corg_api.models.utils import copy_attributes
+from c2corg_api.models import schema, enums
+from c2corg_api.models.utils import copy_attributes, ArrayOfEnum
 from c2corg_api.models.document import (
     ArchiveDocument, Document, get_update_schema, geometry_schema_overrides,
     schema_document_locale, schema_attributes)
-from c2corg_common.attributes import activities
 
 IMAGE_TYPE = 'i'
 
 
 class _ImageMixin(object):
-    activities = Column(
-        Enum(name='activities', inherit_schema=True, *activities),
-        nullable=False)
+
+    activities = Column(ArrayOfEnum(enums.activity_type))
+
+    categories = Column(ArrayOfEnum(enums.image_category))
+
+    image_type = Column(enums.image_type)
+
+    author = Column(String(100))
+
+    # FIXME not sure still needed:
```

Comment:
I guess we still need the "has_svg" flag then, at least to figure out when to display the original SVG and when to display the converted/resized version. Unless we can have resized/cropped SVG versions?

Would be cool but that would mean that all SVG files from v5 must be resized/cropped when migrating the images to v6.